UVicMartletplace/martletplace
# Added auth to getListingsByUser.

# Description
Authentication functionality needs to be added to the getListingByUser feature, so users can see their created listings, regardless of which user they are logged into.

Currently, it is hardcoded to only work with userID = 1.

This PR also fixes the "Edit" button API call not working for some listing views.



Closes #355 

## How to Test

Create a new account on martletplace, and create new listings. Then go to "my listings" and ensure those listings show up.

## Checklist
